resources/assets/components/SignupCard/index.js

```diff
@@ -19,12 +19,29 @@ class SignupCard extends React.Component {
 
     const signupUrl = `/signups/${signup.signup_id}`;
 
+    var campaign_run_start_date = null;
+
+    for (let key in campaign.campaign_runs.current) {
```

Comment:
I think it might be nice to have this in a function defined on the component and then in the rendering you could just do `const campaignRunStartDate = this.getCampaignStartDate(campaign)`